.eslintrc

```diff
@@ -16,8 +16,12 @@ rules:
     - avoid-escape
   semi:
     - error
-    - never
+    - always
```

Comment:
I presume this is is because it is the only way to manage with the existing code, right?